context/app/static/js/components/workspaces/WorkspaceToastSuccessMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+function WorkspaceToastSuccessMessage(id: number) {
+  return (
+    <Stack sx={{ maxWidth: '22rem' }}>
```

Comment:
Why this `maxWidth` in particular?

Reply:
This was from the designs - without a maxWidth, the toast stretches across the bottom of the screen and is hard to read. Is there a better method for setting width options like these?

Reply:
Would `fit-content` work? https://developer.mozilla.org/en-US/docs/Web/CSS/max-width#fit-content

Reply:
That's a bit better than the 100% width - it still stretches out further than the designs, because I think the max-content is higher than 20rem. The `<length-percentage> `parameter doesn't seem to change anything, which is a bit odd:

<img width="700" alt="Screenshot 2024-08-23 at 2 05 08 PM" src="https://github.com/user-attachments/assets/4bb29299-263f-48a5-884e-d1286dfc99b3">